### PR TITLE
Update alpine runtime-deps to get lttng-ust from main package feed

### PR DESCRIPTION
**Pull request by MichaelSimons**
1. Update the 3.8 alpine runtime-deps image to pull lttng-ust from the main package feed vs edge.
2. Update the 3.8 and 3.9 alpine runtime-deps Dockerfile formatting to follow the Dockerfile best practices.

### CODE REVIEW

File: tests/Microsoft.DotNet.Docker.Tests/ImageScenarioVerifier.cs [hunk 1]
```diff
@@ -57,9 +57,9 @@ public async Task Execute()
                 if (DockerHelper.IsLinuxContainerModeEnabled)
                 {
                     // Use `sdk` image to publish self contained app and run with `runtime-deps` image
-                    string selfContainedTag = BuildTestAppImage("self_contained_app", appDir, customBuildArgs: $"rid={_imageData.Rid}");
-                    tags.Add(selfContainedTag);
-                    await RunTestAppImage(selfContainedTag, runAsAdmin: runAsAdmin);
+                    // string selfContainedTag = BuildTestAppImage("self_contained_app", appDir, customBuildArgs: $"rid={_imageData.Rid}");
+                    // tags.Add(selfContainedTag);
+                    // await RunTestAppImage(selfContainedTag, runAsAdmin: runAsAdmin);
```
Comment (mthalman) [inline tests/Microsoft.DotNet.Docker.Tests/ImageScenarioVerifier.cs]: Are these changes intentional?

Comment (MichaelSimons) [reply tests/Microsoft.DotNet.Docker.Tests/ImageScenarioVerifier.cs]: No they weren't.  Reverting.
